src/Salt/SiteBundle/Controller/CommentsController.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+namespace Salt\SiteBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Salt\SiteBundle\Entity\Comment;
+use Salt\SiteBundle\Entity\CommentUpvote;
+use Qandidate\Bundle\ToggleBundle\Annotations\Toggle;
+
+/**
+ * @Toggle("comments")
+ */
+class CommentsController extends Controller
+{
+    /**
+     * @Route("/comments", name="create_comment")
+     *
+     * @Method("POST")
+     *
+     * @Security("is_granted('comment')")
+     */
+    public function newAction(Request $request, UserInterface $user)
+    {
+        $comment = new Comment();
+        $em = $this->getDoctrine()->getManager();
+
+        $itemId = $request->request->get('itemId');
+        $itemType = $request->request->get('itemType');
+        $parentId = $request->request->get('parent');
+
+        if ($this->existItem($itemId, $itemType)) {
+            $comment->setContent(trim($request->request->get('content')));
+            $comment->setUser($user);
+            $comment->setFullname($user->getUsername().' - '.$user->getOrg()->getName());
+            $comment->setItem($itemType.':'.$itemId);
+
+            if (!empty($parentId) && filter_var($parentId, FILTER_VALIDATE_INT)) {
+                $parent = $em->getRepository('SaltSiteBundle:Comment')->findById($parentId);
+                $comment->setParent(($parent)?$parentId:null);
+            } else {
+                $comment->setParent(null);
+            }
+
+            $em->persist($comment);
+            $em->flush();
+
+            $response = $this->apiResponse($comment);
+            return $response;
+        }
+
+        return new Response('Item not found', 404);
```

Comment:
I'm going to make the change, although the only thing I need from this response is the status code.